tests/nist/nist_test.sh

```diff
@@ -21,6 +21,65 @@ function test_nist {
 	fi
 }
 
+function test_results_directive {
+	test_dir="$1"
+	run_script="$(cd "${builddir}"; pwd)/run"
```

Comment:
As we have test_common.sh sourced in this file, you can use $OSCAP, instead of defining run_script by yourself.